README.md

```diff
@@ -8,30 +8,160 @@
+
+3. Select the appropriate account type for your development needs.
+
+   ![Choose account type](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-shopify/refs/heads/main/docs/resources/choose-account.png)  
+
+4. Complete the store setup by following the instructions. You can customize store settings as needed.
+
+   ![Setup store](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-shopify/refs/heads/main/docs/resources/setup-shopify-store.png)  
+
+### 4. Create a new app
+
+To start developing on your store, you need to create an app.
+
+1. Under **App and sales channels** in settings, select **Develop apps**.
+
+   ![Apps & sales](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-shopify/refs/heads/main/docs/resources/apps-and-sales-channel.png)
+
+2. Select **Allow custom app development**.
+
+   ![Develop apps](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-shopify/refs/heads/main/docs/resources/develop-apps.png)  
+
+3. Click **Create an app**, enter an app name, and confirm.
+
+   ![Create app](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-shopify/refs/heads/main/docs/resources/create-app.png)  
+
+### 5. Generate an access token
+
+1. Once the app is created, select **Configure Admin API scopes** and apply the necessary scopes for your REST APIs.
+
+   ![Configure API scopes](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-shopify/refs/heads/main/docs/resources/configure-admin-api-scopes.png)  
+
+2. Click **Install app** to generate an Admin API access token.
+
+   ![Add access token](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-shopify/refs/heads/main/docs/resources/add-access-token.png)
+
+3. Click **Reveal token once** to copy and save the token securely. This token will be required for authentication when using the Ballerina Shopify connector.
+
+   ![Reveal token](https://raw.githubusercontent.com/ballerina-platform/module-ballerinax-shopify/refs/heads/main/docs/resources/reveal-access-token.png)  
+
+Your development store is now ready, and you can integrate it with Shopify using the Ballerina Shopify connector.
 
 ## Quickstart
 
-[//]: # (TODO: Add a quickstart guide to demonstrate a basic functionality of the module, including sample code snippets.)
+To use the Shopify connector in your Ballerina project, modify the `.bal` file as follows.
+
+### Step 1: Import the module
+
+Import the `ballerinax/shopify` module into your Ballerina project.
+
+```ballerina
+import ballerinax/shopify;
+```
+
+### Step 2: Instantiate a new connector
+
+Create a `shopify:Client` instance with necessary configurations.
+
+```ballerina
+configurable string apiVersion = ?;
+configurable string accessToken = ?;
+configurable string serviceUrl = ?;
+
+shopify:Client shopify = check new (config = {
+    followRedirects: {
+        enabled: true
+    },
+    apiVersion,
```

Comment:
Why do we need to provide the API version as a parameter here, if we are only supporting one version of the API?